### ISSUE TITLE
Headers

### ISSUE THREAD

**Issue by Robert1402** [GavickPro/Magazine-Free-Joomla-Template#3]
The headers are all in h1 size. 
How can I change it?

**Comment by bkrztuk** [comment]
Every single header element you can change in proper view overrided by template. Could you tell me which headers you want to change?

**Comment by Robert1402** [comment]
see attachement, but I would like to change the size of headers in all artikel.
![magazin](https://cloud.githubusercontent.com/assets/6908425/2835616/4a30b2a4-cfec-11e3-985c-08e490fd1dd3.png)

**Comment by bkrztuk** [comment]
As I see it is default article view so you can modify this header size in root/templates/gk_magazine/html/com_content/article/default.php file.

**Comment by Robert1402** [comment]
I changed there h1 to h3 but the headers has the same size?
What did I wrong?

**Comment by bkrztuk** [comment]
Could you give me your site URL ? It is always easier to check CSS/HTML code on live site.

**Comment by Robert1402** [comment]
http://www.test.diamantbericht.de/

Thanks.

**Comment by Robert1402** [comment]
http://www.test.diamantbericht.de/index.php/k2-tags/template/articles/news

All headers are to big.

**Comment by bkrztuk** [comment]
This is not single K2 item view but display items by tag. You want to just replace h1 with H3 or decrease size of H1 ? In my opinion you just should make some CSS correction for H1 tag instead of replacing H1 to H3 in every template single file. Please add this code to _override.css_ file in root/templates/gk_template_name/css directory :

```
article header h1 { font-size: 30px !important}
```

and then just remember to enable "Use override CSS" option in template advanced settings tab.

**Comment by Robert1402** [comment]
Super, thanks. It works.
Only 1 problem more: Nor I have  space between header and text.
And between the rows in the text.
![image](https://cloud.githubusercontent.com/assets/6908425/2888019/cc5576fe-d503-11e3-9b59-b543e3b2c101.png)

**Comment by Robert1402** [comment]
Sorry to distuerb you again, but if I click on the new small header, on the next page I get again big header.
![image](https://cloud.githubusercontent.com/assets/6908425/2889120/de0c975c-d516-11e3-819c-622e66df1873.png)

**Comment by bkrztuk** [comment]
You get bigger header because you've modified H1 tag to H3 and now you need to apply the same rule for H3 element by adding code : 

```
article header h3 { font-size: 15px !important}
```

**Comment by Robert1402** [comment]
Many thanks.
What about big space between header and text?

**Comment by Robert1402** [comment]
Many thanks.
What about big space between header and text?
Can you help me?

**Comment by bkrztuk** [comment]
If you want to customize template some base knowledge of Firebug and other tool will be necessary, please read more here » → [How to change space, colors and font size ? – GavickPro Documentation](http://www.gavick.com/documentation/joomla-templates/customizing-joomla-template/how-to-change-paddings-margins-font-size-etc/)

To decrease space please use this code : 

```
article header { margin: 0}
```

**Comment by Robert1402** [comment]
Last question:
How can I change the size of logo?
![image](https://cloud.githubusercontent.com/assets/6908425/2926846/4e5524aa-d762-11e3-982e-9af18dd15d4e.png)

**Comment by bkrztuk** [comment]
Please add this code to **override.css** file in root/templates/gk_template_name/css directory :

```
#gkLogo.cssLogo { width: *value*; height: *value* }
```

and then just remember to enable "Use override CSS" option in template advanced settings tab.

**Comment by Robert1402** [comment]
I add
#gkLogo.cssLogo { width: 400; height: 100}

but nothing change.

**Comment by bkrztuk** [comment]
You forget about the unit, the best way is to use **pixels**  like 400px or 100px

**Comment by Robert1402** [comment]
Perfect! It works!
But I have another problem: I imported my mysql-files to the new website magazin, which I made with GK Quckstart. But they have another names started j25_ and yours with cnjrf_ 
Shoul I rename all  in cnjrf_ ?
Maybe you have a better idea?

**Comment by aml987** [comment]
@Robert1402  try to edit configuration.php file, you can find it in the main root:
on line 19 :  public $dbprefix = 'nal4t_';

**Comment by Robert1402** [comment]
Thanks, aml987 what shuld I do with configuration.php file?

**Comment by aml987** [comment]
Open it then try to replace $dbprefix = 'j25_';   by  $dbprefix = 'cnjrf_';

**Comment by Robert1402** [comment]
In my  configuration.php is already  $dbprefix = 'cnjrf_' and if I change it to 'j25_' get error!

**Comment by aml987** [comment]
what is the perfix of your database? you can get it from phpMyadmin ...

**Comment by Robert1402** [comment]
I have 2 prefix: 'cnjrf_'  and 'j25_' in database.

**Comment by bkrztuk** [comment]
It meana that you have two installations in this one database

**Comment by Robert1402** [comment]
No, which I made with GK Quckstart the website. All files in mysql statrts with cnjrf_.
Than I imported all fils from my old mysql in the same database, because I need the articel, user, categorys etc. But those files-names started j25_ and yours with cnjrf_
Shoul I rename all in cnjrf_ ?

**Comment by Robert1402** [comment]
![image](https://cloud.githubusercontent.com/assets/6908425/3078166/f77a2148-e45f-11e3-86aa-4d785045f89f.png)
How can I get the voting-stars in this module?

**Comment by bkrztuk** [comment]
You firstly need to update module to latest version and use %STARS as a parameter in INFO area like on image here » http://cl.ly/image/1b0c1I1d2m3y

**Comment by Robert1402** [comment]
I get:

Von: Bartłomiej Krztuk [mailto:notifications@github.com] 
Gesendet: Donnerstag, 29. Mai 2014 11:06
An: GavickPro/Magazine-Free-Joomla-Template
Cc: Robert1402
Betreff: Re: [Magazine-Free-Joomla-Template] Headers (#3)

You firstly need to update module to latest version and use %STARS as a parameter in INFO area like on image here » http://cl.ly/image/1b0c1I1d2m3y

**Comment by Robert1402** [comment]
Update: you mean News pro 4 to news pro 5?
How can I update the moduls? 
But if I install them new, I have to make all the changes again!

**Comment by bkrztuk** [comment]
%STARS option is available only in News Show Pro GK5, to update module you just need to download latest package from gavick.com website and install via extension manager like every single Joomla! extension.
